src/alto-ui/Aside/__tests__/Aside.test.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import Aside from '../Aside';
+import CloseButton from '../../CloseButton';
+
+jest.mock('../../CloseButton', () => {
+  return () => {
+    return <div className="CloseButton" />;
+  };
+});
+
+describe('<Aside />', () => {
+
+  const getWrapper = props => {
+    console.log(props);
+    return mount(<Aside {...props} />);
+  };
+
+  it('should return null if show props is falsy', () => {
+    const props = {
+      show: false,
+    };
+    const wrapper = getWrapper(props);
+
+    expect(wrapper.find('.Aside__header')).toHaveLength(0);
+    expect(wrapper.find('.Aside__content').exists()).toBe(false);
+  });
+
+  it('should render if show prop is truthy', () => {
+    const wrapper = getWrapper();
+
+    expect(wrapper.find('.Aside__header')).toHaveLength(1);
+    expect(wrapper.find('.Aside__content').exists()).toBe(true);
+  });
+
+  it('should support show and a11yCloseLabel prop', () => {
+    const props = {
+      a11yCloseLabel: 'test',
+    };
+    const wrapper = getWrapper(props);
+
+    expect(wrapper.find(Aside).prop('show')).toBe(true);
```

Comment:
This test tests nothing!
This test should be on `<CloseButton />` not for main component.
```
const { open, a11yCloseLabel } = wrapper.find(CloseButton).props();
expect(open).toBe(true);
expect(a11yCloseLabel).toBe('test');
```

Reply:
Refactored this test to test `<CloseButton/>` props